_ont/ont-huawei-ma5671a-root.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: Root Procedure for Huawei MA5671A
+title: Root Procedure for Huawei MA5671A (V2)
```

Comment:
Where is the root procedure V1?

Reply:
currently removed, perhaps it can be restored instead of showing a link to a commit?